.github/workflows/build.yml

```diff
@@ -270,12 +300,19 @@ jobs:
       - platforms
     runs-on: ubuntu-latest
     container:
-      image: centos:${{ matrix.node < 18 && '7' || '8' }}
+      image: ${{ matrix.node < 18 && 'centos:7' || (matrix.node < 25 && 'centos:8' || 'rockylinux:9') }}
       volumes:
         - /node20217:/node20217:rw,rshared
         - /node20217:/__e/node20:ro,rshared
     name: centos-test-${{ matrix.node }}
     steps:
+      # Rocky Linux 9 doesn't seem to come with xz preinstalled, and on x64 it
+      # also lacks libatomic which Node.js 25 needs. (Weirdly, it does come with
+      # it on arm64.)
+      - name: Install libraries for Rocky Linux 9
```

Comment:
Since Node 25 itself supports and runs on CentOS 8, should we use it to match their level of support?

From the [doc](https://github.com/nodejs/node/blob/main/BUILDING.md#official-binary-platforms-and-toolchains) it mentions that it does support it but that libatomic needs to be installed, which a customer could do: `RHEL 8 with Clang 19.1 and gcc-toolset-14-libatomic-devel`